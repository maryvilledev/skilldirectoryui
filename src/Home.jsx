import React, { Component } from 'react';

function Home(props) {
  return (
    <h1>Skill Directory Home</h1>
  );
}

export default Home