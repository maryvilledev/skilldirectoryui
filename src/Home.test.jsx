import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Home from './Home';

describe('<Home />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Home />, div);
  });

  it('matches the stored snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});