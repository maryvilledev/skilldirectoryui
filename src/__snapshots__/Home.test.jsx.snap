exports[`<Home /> matches the stored snapshot 1`] = `
<div>
  <h1>
    Skill Directory Home
  </h1>
  <h3>
    Team Members: 
    <a
      href="/team" />
  </h3>
  <h3>
    Unique Skills: 
    <a
      href="/skills" />
  </h3>
  <ul />
</div>
`;
